app/models/location.rb

```diff
@@ -1,7 +1,17 @@
 class Location < ActiveRecord::Base
+  SUPPORTED_ZIPCODES = %w[80221].freeze
+
   belongs_to :user, touch: true
 
   validates :address, presence: true
   validates :user, presence: true
   validates :zipcode, presence: true
+
+  def supported?
+    SUPPORTED_ZIPCODES.include?(zipcode)
+  end
+
+  def zipcode=(zipcode)
+    super(zipcode.to_s.strip)
```

Comment:
if you do it in the controller, then the model doesn't have to care that it needs to do anything with the zipcode, it just knows it's always going to get a zipcode in the correct format. Also, in the controller if you have a malformed zipcode you can immediately act on it and tell the user. Not sure if @joshuaclayton was thinking the same thing, but those are my thoughts.